infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -79,6 +84,13 @@
     @Getter
     private ObservableValue rxMessageCount = new ObservableValue(rxMessageCounter);
 
+    /*
+     * Periodic Thread which reads the sequence number of entries processed
+     */
+    private ScheduledExecutorService lastReplicatedEntryPoller = Executors.newSingleThreadScheduledExecutor();
```

Comment:
Do we need an executor service for the standby? Cause it would only be updated to true (or 100) once the snapshot is applied, right? so we can just do it directly, instead of having a thread for this every 15 seconds... anyways its not changing.